### ISSUE TITLE
Issue 3 : Adding some content in file

### ISSUE THREAD

**Issue by afreenabegum** [marshadkhn/GitPractice#70]
In previous Issue, I have added files into repository via Gihub directly, was unable to track changes through git at that time.
Now, I want to try adding files through Git, want to push files through commands.

Let me give a try. So, please assign me this issue @marshadkhn .

**Comment by marshadkhn** [comment]
Don't use Github GUI please it will not help you

**Comment by afreenabegum** [comment]
> Don't use Github GUI please it will not help you

Yes, tried with Git commands yesterday. But unable to complete that. Will try today.

**Comment by marshadkhn** [comment]
> > Don't use Github GUI please it will not help you
> 
> Yes, tried with Git commands yesterday. But unable to complete that. Will try today.

Sure
